src/process.cr

```diff
@@ -240,14 +244,24 @@ class Process
           fork_error || error,
           chdir
         )
+      rescue ex : Errno
+        value = ex.errno
+        child_sock.write(Bytes.new(pointerof(value).as(UInt8*), 4))
       rescue ex
         ex.inspect_with_backtrace STDERR
-        LibC._exit 127 # TODO: remove after 0.19
       ensure
         LibC._exit 127
       end
     end
 
+    child_sock.close
+    slice = Bytes.new(4)
+    if parent_sock.read(slice) == 4
+      parent_sock.close
+      Errno.value = slice.pointer(4).as(Int32*).value
```

Comment:
Shouldn't you just pass the value to `Errno`'s constructor instead of setting `Errno.value`?